best-practices/environments.md

```diff
@@ -0,0 +1,12 @@
+# Environments
+
+This document clarifies the terminology we use to talk about deployment environments.
+
+<dl>
+  <dt>Production (or "prod") </dt>
+  <dd>The production environment is the one used by most end-users. These environments tend to be the most robustly provisioned, and also to hold data that we care about, *i.e.*, that gets preserved to SDR and backed up regularly.</dd>
+  <dt>Staging (or "stage") </dt>
+  <dd>The staging environment mimics production closely, and serves as a last step in testing a feature or set of features against production-like data after the work has undergone initial (user acceptance/quality assurance) testing and it has been merged.</dd>
```

Comment:
production-like environment.

There is always confusion re:   staging *code* to run it through its paces, and staging *data* to run it through its paces.   Both of these need environments that closely mimic production.  Sometimes they can share an environment, but often not.